webapp/sources/scala-ldap/src/main/scala/com/normation/ldap/sdk/LDAPIOResult.scala

```diff
@@ -34,7 +34,8 @@ sealed trait LDAPRudderError extends RudderError {
 object LDAPRudderError {
   // errors due to some LDAPException
   final case class BackendException(msg: String, cause: Throwable)  extends LDAPRudderError {
-    override def fullMsg: String = super.fullMsg + s"; cause was: ${RudderError.formatException(cause)}"
+    override def fullMsg: String = super.fullMsg + s"; cause was: ${RudderError.formatException(cause)} ;" +
+      s"you probably need to increase value ldap.maxPoolSize property in configuration file, please check the documentation"
```

Comment:
maybe we can put a link to the documentation ? maybe the embedded one ?

Reply:
i'm unsure we will effectively maintain this link in an error message :/

Reply:
I think it's better with the link. It's better for the user, and even if it becomes incorrect at some point, it helps the user going at the right place. 
Not sure if we have the base url for the local doc though.